src/org/labkey/test/components/ui/Pager.java

```diff
@@ -160,7 +151,7 @@ public int total()
      * if the pager does not show a total, it indicates that there are no more to the set than are shown on the current page.
      * use this to understand whether or not there are more pages of information (which
      * could be the whole set, or a filtered range of it, or both).
-     * @return
+     * @return <code>true</code> if pager shows a "total"
      */
     public boolean hasTotal()
```

Comment:
I'm going to make an argument to remove this method. It doesn't look like it is used (that should be reason enough), and second elementCache().counts has a time out of 4 seconds. If I understand the code correctly that means to get a false from this method you have to wait 4 seconds.